skill-tree/src/test/java/com/RDS/skilltree/EndorsementsIntegrationTests.java

```diff
@@ -37,7 +37,7 @@ public EndorsementsIntegrationTests(UserService userService, UserRepository user
     }
 
     @BeforeEach
-    private void addData() throws MalformedURLException {
+    public void addData() throws MalformedURLException {
         user = userService.createUser(UserDRO.builder()
                 .role(UserRole.MEMBER)
                 .rdsUserId("p6Bo61VEClhtVdwW0ihg")
```

Comment:
I do see some hard coded url in line #46  can you remove that

Reply:
This was not my change, @bhtibrewal - can you please let me know what I can replace the hardcoded URL in line 46 with?